packages/dma-contracts/tasks/service-registry/index.ts

```diff
@@ -0,0 +1,194 @@
+import { getConfigByNetwork } from '@deploy-configurations/configs'
+import { ConfigEntry, SystemConfig } from '@deploy-configurations/types/deployment-config'
+import { Network } from '@deploy-configurations/types/network'
+import { task } from 'hardhat/config'
+
+import {
+  getPropertyFromPath,
+  getServiceRegistry,
+  isInvalidAddress,
+  ServiceRegistryMaybe,
+} from '../common'
+
+function showLocalConfig(configPath: string, network: string) {
+  const config: SystemConfig = getConfigByNetwork(network as Network) as SystemConfig
+
+  const property = configPath === 'all' ? config : getPropertyFromPath(config, configPath)
+
+  if (!property) {
+    throw new Error(`Property ${configPath} not found`)
+  }
+
+  console.log('\n====== LOCAL CONFIG ======')
+  console.log(`Network: ${network}`)
+  console.log(`Path: ${configPath}`)
+  console.log('==============================\n')
+  console.log(property)
+  console.log('\n==============================\n')
+}
+
+async function showRemoteConfig(configPath: string, network: string, ethers: any) {
+  const config: SystemConfig = getConfigByNetwork(network as Network) as SystemConfig
+
+  const serviceRegistry: ServiceRegistryMaybe = await getServiceRegistry(ethers.provider, config)
+  if (!serviceRegistry) {
+    console.log('ServiceRegistry not deployed, cannot fetch values')
+    return
+  }
+
+  const property = getPropertyFromPath(config, configPath)
+
+  if (!property) {
+    throw new Error(`Property ${configPath} not found`)
+  }
+
+  console.log('\n====== REMOTE CONFIG ======')
+  console.log(`Network: ${network}`)
+  console.log(`Path: ${configPath}`)
+  console.log(`ServiceRegistry: ${serviceRegistry.address}`)
+  console.log('==============================\n')
+
+  for (const serviceName of Object.keys(property)) {
+    const service = property[serviceName] as ConfigEntry
+
+    if (!service.serviceRegistryName) {
+      console.log(`[✅][LOCAL CONFIG] ${service.name}: ${service.address}`)
```

Comment:
maybe indicate here that its `not required in SR` or something similar. `LOCAL CONFIG` got me confused so I had to cehck what does taht mean :)

Reply:
This is a tricky one to summarize in the label. But I've added `NOT REQUIRED`, do you think it is better?